shared/templates/audit_rules_syscall_events/template.yml

```diff
@@ -1,3 +1,7 @@
+<<<<<<< HEAD
+=======
```

Comment:
Please remove these lines. This appears to be part of why the build is failing on this branch.

Reply:
Hello @Mab879 - I have removed them.